src/components/CallHistoryLastUpdatedStatus.tsx

```diff
@@ -18,14 +18,24 @@ const CallHistoryLastUpdatedStatus = (): ReactElement => {
         });
     }, []);
 
+    const DisplayResult = () => {
+        const date = mirBstDateFormatter(reportStatus, true);
```

Comment:
While we're on the subject of naming, `reportStatus` doesn't seem like a great name for something that is actually a date?